### DISCUSSION TITLE
any plans to scrape/include rest of nutrition info

### DISCUSSION THREAD

**Discussion by sdetweil** [mealie-recipes/mealie#4071] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

calories,fat,protein, etc

### Please provide a concise description of the feature that would resolve your issue.

many recipes list these attributes, but theyvarent saved/shown in mealie

### Please consider and list out some caveats or tradeoffs made in your design decision

have nothing at this time

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
There are already supported in Mealie:
![image](https://github.com/user-attachments/assets/0860a3d4-4ed3-48b1-be21-a3d7c91f6e66)

If you're not seeing them, make sure you have nutrition info toggled on:
![image](https://github.com/user-attachments/assets/cef9897c-a703-471b-96a4-5a0f2e1987e7)

If you want it to be on by default, you can manage that in your group settings

**Comment by Kuchenpirat** [comment]
This is scraped by mealie if included in the recipe data.
You have to enable it either on a recipe by recipe basis or on via the group settings for all recipes.

**Reply by Kuchenpirat** [reply]
@sdetweil For the future, please to a quick basic spelling check of things you want other people to read/spend their time on. 
It does not need to perfect, but try to catch the verry obvious stuff.

**Comment by sdetweil** [comment]
i edited it almost immediately. this dang new phone is killing me

**Reply by Kuchenpirat** [reply]
no worries, I feel the same all the time ;)
